src/views/PerWorkPlanForm/index.tsx

```diff
@@ -263,26 +264,27 @@ export function Component() {
     const componentResponseError = getErrorObject(error?.prioritized_action_responses);
     const customComponentError = getErrorObject(error?.additional_action_responses);
 
-    const readOnlyMode = statusResponse?.phase !== PER_PHASE_WORKPLAN;
+    const readOnlyMode = isNotDefined(statusResponse)
+        || isNotDefined(statusResponse.phase)
+        || statusResponse.phase < PER_PHASE_WORKPLAN;
 
     return (
         <Container
             className={styles.perWorkPlanForm}
             childrenContainerClassName={styles.content}
-            footerActions={(
+            footerActions={value.is_draft !== false && (
```

Comment:
What is value.is_draft is undefined? Shouldn't it be treated as false?

Reply:
No, `is_draft` will be undefined when creating the workplan

Reply:
We only hide submit button if the form is already submitted